tests/teams/test_hidden_team_scores.py

```diff
@@ -0,0 +1,98 @@
+from tests.helpers import (
+    create_ctfd,
+    destroy_ctfd,
+    login_as_user,
+    gen_challenge,
+    gen_flag,
+    gen_user,
+    gen_team,
+)
+from CTFd.models import Teams
+
+from CTFd.utils.scores import get_standings, get_team_standings
+
+
+def setup_test(app):
+    user1 = gen_user(app.db, name="user1", email="user1@ctfd.io")
+    team1 = gen_team(app.db, name="team1", email="team1@ctfd.io")
+    user1.team_id = team1.id
+    team1.members.append(user1)
+    team1.hidden = True
+
+    user2 = gen_user(app.db, name="user2", email="user2@ctfd.io")
+    team2 = gen_team(app.db, name="team2", email="team2@ctfd.io")
+    user2.team_id = team2.id
+    team2.members.append(user2)
+
+    gen_challenge(app.db)
+    gen_flag(app.db, 1)
+
+    app.db.session.commit()
+
+    with login_as_user(app, name="user1") as client:
+        flag = {"challenge_id": 1, "submission": "flag"}
+        client.post("/api/v1/challenges/attempt", json=flag)
+
+    with login_as_user(app, name="user2") as client:
+        flag = {"challenge_id": 1, "submission": "flag"}
+        client.post("/api/v1/challenges/attempt", json=flag)
+
+
+def test_standings():
+    app = create_ctfd(user_mode="teams")
+
+    with app.app_context():
+        setup_test(app)
+
+        standings = get_standings()
+
+        assert standings[0][2] == "team2"
```

Comment:
This is indeed better. I have changed it to using the attributes as you suggested.

I had done it this way because that's what the test in tests/teams/test_scoreboard.py is using. I haven't changed it in that file to keep this PR from becoming too bloated.